app/src/main/java/com/randomappsinc/studentpicker/utils/SpeechUtil.java

```diff
@@ -0,0 +1,45 @@
+package com.randomappsinc.studentpicker.utils;
+
+import android.app.Activity;
+import android.content.ActivityNotFoundException;
+import android.content.Context;
+import android.content.Intent;
+import android.speech.RecognizerIntent;
+
+import com.randomappsinc.studentpicker.R;
+
+import java.util.List;
+import java.util.Locale;
+
+import static android.app.Activity.RESULT_OK;
+
+public class SpeechUtil {
+
+    public static final int SPEECH_REQUEST_CODE = 1;
```

Comment:
We should let the callers customize this code IMHO. If the caller is using 1 for something else, we will have bugs :(